tests/runtests.py

```diff
@@ -46,6 +46,12 @@
 # Make deprecation warnings errors to ensure no usage of deprecated features.
 warnings.simplefilter("error", RemovedInDjango60Warning)
 warnings.simplefilter("error", RemovedInDjango61Warning)
+# Ignore this message as the test suite uses the default
+warnings.filterwarnings(
+    "ignore",
+    message="Multiline tags in templates will become the default in Django 6.0",
+    category=RemovedInDjango60Warning,
+)
```

Comment:
I think we should remove this
We need to have a test to cover the deprecation warning, see ec7044c706f48f5ab3d9e4c35e4078b9f9dcaaf2 as an example
We also need to document the deprecation in `docs/internals/deprecation.txt`

Reply:
Ah, yea I had trouble doing that. I couldn't figure out where to change the test suite settings to turn on multiline by default, so this is the best I could do.